content/blog/deploy-dotnet-tap-aws/index.md

```diff
@@ -0,0 +1,29 @@
+---
+date: '2022-10-7'
+lastmod: '2022-10-7'

```

Comment:
```suggestion
date: '2022-10-07'
lastmod: '2022-10-07'
```

I actually didn't know this until now, but looks like Hugo is super picky with date formatting.